macros/src/main/scala/MacroCompiler.scala

```diff
@@ -834,36 +834,24 @@ object MacroCompiler extends App {
             )
           ))
         )
-        // Append a NoDCEAnnotation to avoid dead code elimination removing the non-parent SRAMs
-        val state = CircuitState(circuit, HighForm, annotations :+ NoDCEAnnotation)
-
-        // Run the compiler.
-        val result = new MacroCompiler().compileAndEmit(state)
-
-        // Write output FIRRTL file.
-        params.get(Firrtl) match {
```

Comment:
Can we file an issue about this removal to discuss/keep track of who needed it.

Reply:
Good idea, I put that in #81